packages/frontend/src/components/dynamic/album/AlbumDetails.tsx

```diff
@@ -23,32 +23,43 @@ const AlbumDetails: React.FC<AlbumDetailsProps> = ({
           className={`${gridCol} grid h-[300px] w-full gap-[1px] overflow-hidden rounded md:h-[200px] md:w-[200px] lg:h-[270px] lg:w-[300px]`}
         >
           {albumCoverImages?.map((items, i) => {
-            const originalYoutubeUrl = items.original_youtube_url ?? "";
-            const youtubeVideoId = getYoutubeVideoId(originalYoutubeUrl);
-
-            if (youtubeVideoId) {
-              const thumbnailUrl = `https://i.ytimg.com/vi/${youtubeVideoId}/hqdefault.jpg`;
-
-              return (
-                <div
-                  key={i}
-                  className="flex items-center justify-center overflow-hidden"
-                >
-                  <div className="relative h-[135%] w-full">
-                    <Image
-                      src={thumbnailUrl}
-                      alt={items.name!}
-                      fill={true}
-                      priority={true}
-                      sizes="(max-width: 768px) 100vw, (max-width: 1200px) 50vw, 33vw"
-                      className="object-cover"
-                    />
-                  </div>
-                </div>
+            let thumbnailUrl = "";
+            if (items.original_youtube_url) {
+              const youtubeVideoId = getYoutubeVideoId(
+                items.original_youtube_url,
               );
+              thumbnailUrl = `https://i.ytimg.com/vi/${youtubeVideoId}/hqdefault.jpg`;
+              console.log("exist");
             }
+
+            return (
+              <div
+                key={i}
+                className={`${
+                  thumbnailUrl === "" && "border-2"
+                } flex items-center justify-center overflow-hidden`}
+              >
+                <div className="relative h-[135%] w-full">
+                  <Image
+                    src={
+                      thumbnailUrl !== "" ? thumbnailUrl : "/no-album-cover.svg"
```

Comment:
easier way to write this is `thumbnailUrl || /no-album-cover.svg` but its alright, its just shorter this way